CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Changelog
 
+## v0.31.4
+
+*April 12th, 2019*
+
+This release fixes a regression from v0.31.3 which used the peer's `SocketAddr` to add the peer to
```

Comment:
😞

Reply:
Do you want me to change sth here? Or is this 😞just an expression of sadness about that regression? :-D

Again, this is probably sth we could have catched by running more extensive (actual) autm. testing/testnet, or, what @greg-szabo and @mircea-c call automated QA.

Reply:
> Do you want me to change sth here? 
no

Reply:
Can we introduce emoji prefixes for the different entries? :cry: for regressions for example. :p

Reply:
Lol great idea!